examples/bigquery/main.tf

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
We should have a README for each example which includes a brief explanation of what it demonstrates/does and the autogenerated docs on inputs/outputs.

Reply:
Correct, will do.

Reply:
Is the main README at examples/README.md not covering the 3 examples docs already ?

Reply:
I think it's helpful to have a brief summary in each example. The main additional benefit is that the README can be a destination for example input/output autogenerated docs.